### ISSUE TITLE
smplx_param.pkl are unaligned with mesh_smplx.obj

### ISSUE THREAD

**Issue by Tessantess** [ytrock/THuman2.0-Dataset#9]
The smplx mesh generated by smplx_param.pkl is unaligned with mesh_smplx.obj. Here is the code: 
```
param = np.load(param_path, allow_pickle=True)
param = {k: torch.as_tensor(v) for k, v in param.items()}
smpl_out = self.smpl_model(shape_params=param['betas'],
                                   expression_params=param['expression'],
                                   body_pose=param['body_pose'],
                                   global_pose=param['global_orient'],
                                   jaw_pose=param['jaw_pose'],
                                   left_hand_pose=param['left_hand_pose'],
                                   right_hand_pose=param['right_hand_pose'],
                                   leye_pose=param['leye_pose'],
                                   reye_pose=param['reye_pose'])

smpl_verts = (smpl_out.vertices[0] * param['scale'] + param['translation'].view(1, 3)).detach().cpu().numpy()
```

and the generated results of ID 0000 with released smplx mesh: 
![image](https://user-images.githubusercontent.com/45743512/167365628-faa81787-3497-4ab4-a603-add894515f0d.png)

**Comment by ytrock** [comment]
Hi Tingting,

Do you use the officially released SMPL-X model for generating the mesh?

**Comment by Tessantess** [comment]
> Hi Tingting,
> 
> Do you use the officially released SMPL-X model for generating the mesh?

Yes, I download it from the official link.

**Comment by YuliangXiu** [comment]
@Tessantess, please check [here](https://github.com/YuliangXiu/ICON/blob/master/docs/dataset.md).

**Comment by wangsen1312** [comment]
@ytrock Same problem here. BTW: how to translate the hand_pose (12 dim in your offered params) to the default one (25*3 in SMPLX default).

**Comment by ytrock** [comment]
@wangsen1312 , the provided 12 dim parameters is the PCA parameters of the hand pose, so please use the corresponding interface for generating the mesh.

**Comment by wangsen1312** [comment]
Thank you! Everything goes perfect now. @ytrock

**Comment by Tessantess** [comment]
> Hi Tingting,
> 
> Do you use the officially released SMPL-X model for generating the mesh?

The results gets right  after using specific gender parameters. Could you please release the corresponding gender list of all subjects?
